pkg/webhook/servicecatalog/clusterservicebroker/validation/handler_access_test.go

```diff
@@ -0,0 +1,290 @@
+		object    []byte
+	}{
+		"Request for Create ClusterServiceBroker should be allowed": {
+			admissionv1beta1.Create,
+			[]byte(`{
+  				"apiVersion": "servicecatalog.k8s.io/v1beta1",
+  				"kind": "ClusterServiceBroker",
+  				"metadata": {
+				  "finalizers": ["kubernetes-incubator/service-catalog"],
+  				  "creationTimestamp": null,
+  				  "name": "test-broker"
+  				},
+  				"spec": {
+				  "url": "http://test-broker.local"
+  				}
+			}`),
+		},
+		"Request for Update ClusterServiceBroker should be allowed": {
+			admissionv1beta1.Update,
+			[]byte(`{
+  				"apiVersion": "servicecatalog.k8s.io/v1beta1",
+  				"kind": "ClusterServiceBroker",
+  				"metadata": {
+				  "finalizers": ["kubernetes-incubator/service-catalog"],
+  				  "creationTimestamp": null,
+  				  "name": "test-broker"
+  				},
+  				"spec": {
+				  "url": "http://test-broker.local"
+  				}
+			}`),
+		},
+		"Request for Create ClusterServiceBroker with AuthInfo should be allowed": {
+			admissionv1beta1.Create,
+			[]byte(`{
+  				"apiVersion": "servicecatalog.k8s.io/v1beta1",
+  				"kind": "ClusterServiceBroker",
+  				"metadata": {
+				  "finalizers": ["kubernetes-incubator/service-catalog"],
+  				  "creationTimestamp": null,
+  				  "name": "test-broker"
+  				},
+  				"spec": {
+				  "url": "http://test-broker.local",
+				  "authInfo": {
+    			    "basic": {
+      				  "secretRef": {
+        			    "namespace": "test-handler",
+						"name": "` + AllowedSecretName + `"
+					  }
+					}
+				  }
+  				}
+			}`),
+		},
+		"Request for Update ClusterServiceBroker with AuthInfo should be allowed": {
+			admissionv1beta1.Update,
+			[]byte(`{
+  				"apiVersion": "servicecatalog.k8s.io/v1beta1",
+  				"kind": "ClusterServiceBroker",
+  				"metadata": {
+				  "finalizers": ["kubernetes-incubator/service-catalog"],
+  				  "creationTimestamp": null,
+  				  "name": "test-broker"
+  				},
+  				"spec": {
+				  "url": "http://test-broker.local",
+				  "authInfo": {
+    			    "bearer": {
+      				  "secretRef": {
+        			    "namespace": "test-handler",
+						"name": "` + AllowedSecretName + `"
+					  }
+					}
+				  }
+				}
+			}`),
+		},
+	}
+
+	for desc, test := range tests {
+		t.Run(desc, func(t *testing.T) {
+			// Given
+			handler := validation.AdmissionHandler{}
+			handler.CreateValidators = []validation.Validator{&validation.AccessToBroker{}}
+			handler.UpdateValidators = []validation.Validator{&validation.AccessToBroker{}}
+
+			err := handler.InjectDecoder(decoder)
+			require.NoError(t, err)
+			err = handler.InjectClient(&fakedClient{})
+			require.NoError(t, err)
+
+			request.AdmissionRequest.Operation = test.operation
+			request.AdmissionRequest.Object.Raw = test.object
+
+			// When
+			response := handler.Handle(context.TODO(), request)
+
+			// Then
+			assert.True(t, response.AdmissionResponse.Allowed)
+		})
+	}
+}
+
+func TestAdmissionHandlerAccessToBrokerDenied(t *testing.T) {
+	//Given
```

Comment:
```suggestion
	// given
```

I know it's nazi, but let's have a common pattern. In other pkg we using lowercase and space after `//`. Please unify that

Reply:
it is